tests/simulator/test_restarts.py

```diff
@@ -205,7 +219,16 @@ def count_levels_and_patches(qty):
                                     f"FAILED domain particles at time {time} {ilvl} {patch1.box} {patch0.box}"
                                 )
                         else:
-                            np.testing.assert_equal(pd0.dataset[:], pd1.dataset[:])
+                            if cpp.mpi_size() == 1:
+                                np.testing.assert_equal(pd0.dataset[:], pd1.dataset[:])
+                            else:
+                                # rank order for shared nodes not guaranteed
+                                np.testing.assert_allclose(
+                                    pd0.dataset[:],
+                                    pd1.dataset[:],
+                                    atol=1e-14,  # 1e-15 fails empirically
```

Comment:
is flakey, could be 1e-12/13?

Reply:
[still flakey, fails sometimes as is](https://hephaistos.lpp.polytechnique.fr/teamcity/buildConfiguration/Phare_Phare_BuildGithubPrClang/81379?buildTab=log&linesState=4939&logView=flowAware&focusLine=6784)